admin-dev/themes/default/js/tree.js

```diff
@@ -213,13 +210,13 @@ Tree.prototype =
 
 			$.get(
 				'ajax-tab.php',
-				{controller:'AdminProducts',token:currentToken,action:'getCategoryTree',type:idTree,fullTree:1,selected:selected, inputName:name,useCheckBox:useCheckBox},
+				{ controller:'AdminProducts', token:currentToken, action:'getCategoryTree', type:idTree, fullTree:1, selected:selected, inputName:name, useCheckBox:useCheckBox },
 				function(content)
 				{
 					$('#' + idTree).html(content);
 					organizeTree();
 					$('#' + idTree).tree('init');
-					that.$element.find("label.tree-toggler").each(
+					$(this).$element.find("label.tree-toggler").each(
```

Comment:
hard for me to see here what $(this) refers to

Reply:
It looks like it refers to `"label.tree-toggler, .icon-folder-close, .icon-folder-open"` but the actual behavior refers to the tree, I don't know which is right, I think we need to checkout @crezzur branch in order to review this, overwise you should maybe write a good description to explain it a bit more :)

Reply:
Not sure what the issue is quys? the element $(this) is a jQuery element which holds the data that was clicked.

As you can see the current code already use the element $(this) but puts it into a variable that and thatOne .
In the current version the variable cannot be accessed from the function. Just try it and you will see i am correct.